app/components/forms/events/view/videoroom-form.hbs

```diff
@@ -62,6 +62,11 @@
           @site="vimeo"
           @value={{this.data.stream.url}}
           @onChange={{action (mut this.data.stream.url)}} />
+      {{else if (eq this.data.stream.videoChannel.provider '3cx')}}
+        <Widgets::Forms::LinkField
+          @inputId="url"
+          @value={{this.data.stream.url}}
+          @onChange={{action (mut this.data.stream.url)}} />
```

Comment:
Isn't it same as else?

Reply:
Yes, will fix it. 
But is this correct? Why do we need another channel when it is similar to custom?

Reply:
Custom won't show in iframe